AY2021S2-CS2103-T14-1/tp
# Add tag issue support

## What this does
This PR
1. adds tag issue support

Closes #90 

## How to test
1. Launch SunRez.
2. Execute the `iadd r/10-100 d/Broken light t/2020/01/12 3:30pm s/pending c/furniture g/HIGH` command. You should see the issue added with the tag `HIGH`.
